docs/docarray-v2.md

```diff
@@ -115,12 +113,27 @@ with Deployment(uses=MyExec) as dep:
 ```
 
 
+## Compatible Features
+
+Jina is working to offer full compatibility of the Jina framework with the power of the new DocArray version.
+
+At this moment, these are the features supported if the APIs described in the previous sections are used.
+
+- All the features offered by {ref}`Deployment <deployment>` where a single Executor is served and {ref}`scaled <scale-out>`. This support includes the usage of both
+HTTP and gRPC protocols and the usage of both of them at the same type.
+
+- When combining multiple Deployments in a pipeline using a {ref}`Flow <flow-cookbook>`, multiple limitations exist at this moment:
+
+    - Only gRPC protocol is supported.
+    - Only linear Flows are supported, no topologies using bifurcations can be used at the moment.
+    - There must exist compatibility between the output schema of an Executor and the input schema of the next one.
```

Comment:
IMO this is not a limitation. DocArray v1 did not solve this problem either, v2 just make it explicit. 

We should rather create a note to say that in order for two executor to work in a flow they need to match output/intput schema

Reply:
can you please propose how you would change it? I am happy to merge that instead.